tools/initDevEnv.js

```diff
@@ -134,11 +140,11 @@ InitDevEnv.prototype = {
       `geth --datadir '${gethFolder}'` +
       ` --networkid ${setUpConfig.chain.networkId}` +
       ` --port ${setUpConfig.chain.geth.port}` +
-      ` --mine --minerthreads 1 --targetgaslimit ${setUpConfig.chain.gas} --gasprice 0x3B9ACA00` +
-      ` --rpc --rpcapi eth,net,web3,personal,txpool --rpcaddr ${setUpConfig.chain.geth.host} --rpcport ${
+      ` --fakepow --mine --minerthreads 16 --targetgaslimit ${setUpConfig.chain.gas} --gasprice 0x3B9ACA00` +
```

Comment:
Why `--fakepow` is needed ?

Reply:
if we actually use `--mine` then we should not *actually* mine on Travis nodes ;)

Reply:
`--fakepow  Disables proof-of-work verification`

Two reasons:
    - One has been also brought by Ben. We should not mine.
    - Second was an attempt to cut tests running time. It cuts it by ~2. In this PR I've also set the difficulty to "0x0". Both together, cuts running time for me from 8mins to 3mins. If we want to have different setups, we should at least have configuration options and handle them in initDevEnv.js accordingly to have a faster running time during a development.